vimc-6428: Add parameter, ref and instance info to queue status

r-ash opened the pull request:
- [x] spec.md has been updated or doesn't need to updated

This will add info about parameters, git ref and db instance used in call to run report to the queue status endpoint. This will also add that info to the "queue" portion of the status endpoint too.

Do you think I should use `params` or `parameters` here? I have added `params` to be consistent with run request but the end goal for this is that is is returned from `orderly::orderly_queue_status` and orderly_run_remote uses `parameters` to refer to parameters. I could keep `params` here for consistency with report run endpoint and map to `parameters` in orderly for consistency with orderly interface.

r-ash commented:
@EmmaLRussell @Nicolas-Dolan @LekanAnni Added you all as a reviewer to note the change to schema. This is exposed in orderly web API atm so wanted to check if it caused breakages on your end?

EmmaLRussell reviewed (commented):
Seeing this error in orderly.server on this branch when getting status for a workflow from orderly web. Any ideas?
```
{"level":400,"timestamp":"2022-05-12 09:57:59","logger":"orderly.server","caller":"postserialize","msg":"response GET /v1/workflow/amphisbaenous_stickleback/status/ => 500 (153 bytes)"}
{"level":200,"timestamp":"2022-05-12 09:57:59","logger":"orderly.server","caller":"postserialize","msg":"error","errors":[{"error":"SERVER_ERROR","detail":"Only atomic vectors of length 1 or data frames with 1 row can be unboxed."}],"method":"GET","path":"/v1/workflow/amphisbaenous_stickleback/status/","query":[],"headers":{"accept":"application/json","accept-encoding":"gzip","connection":"Keep-Alive","host":"localhost:8321","user-agent":"okhttp/4.8.1"},"body":"<binary body (0 bytes)>"}
```

This is the workflow:
![image](https://user-images.githubusercontent.com/44669576/168046417-dd0ed1be-677a-4e9d-b904-c266f225877f.png)

r-ash commented:
> Seeing this error in orderly.server on this branch when getting status for a workflow from orderly web. Any ideas?
> 
> ```
> {"level":400,"timestamp":"2022-05-12 09:57:59","logger":"orderly.server","caller":"postserialize","msg":"response GET /v1/workflow/amphisbaenous_stickleback/status/ => 500 (153 bytes)"}
> {"level":200,"timestamp":"2022-05-12 09:57:59","logger":"orderly.server","caller":"postserialize","msg":"error","errors":[{"error":"SERVER_ERROR","detail":"Only atomic vectors of length 1 or data frames with 1 row can be unboxed."}],"method":"GET","path":"/v1/workflow/amphisbaenous_stickleback/status/","query":[],"headers":{"accept":"application/json","accept-encoding":"gzip","connection":"Keep-Alive","host":"localhost:8321","user-agent":"okhttp/4.8.1"},"body":"<binary body (0 bytes)>"}
> ```
> 
> This is the workflow: ![image](https://user-images.githubusercontent.com/44669576/168046417-dd0ed1be-677a-4e9d-b904-c266f225877f.png)

Thanks yes - I hadn't updated the workflow handling of scalars for new return structure. I've updated the test for that and fixed the issue.

EmmaLRussell commented:
This is better now, but still getting errors the first couple of times it fetches status (but then it fixes itself)
![image](https://user-images.githubusercontent.com/44669576/168789466-7680dc02-e124-4191-9062-dc31ef650a21.png)

Same error I think:
```
{"level":400,"timestamp":"2022-05-17 10:16:09","logger":"orderly.server","caller":"postroute","msg":"request GET /v1/workflow/sirenic_danishswedishfarmdog/status/"}
{"level":400,"timestamp":"2022-05-17 10:16:09","logger":"orderly.server","caller":"postserialize","msg":"response GET /v1/workflow/sirenic_danishswedishfarmdog/status/ => 500 (153 bytes)"}
{"level":200,"timestamp":"2022-05-17 10:16:09","logger":"orderly.server","caller":"postserialize","msg":"error","errors":[{"error":"SERVER_ERROR","detail":"Only atomic vectors of length 1 or data frames with 1 row can be unboxed."}],"method":"GET","path":"/v1/workflow/sirenic_danishswedishfarmdog/status/","query":[],"headers":{"accept":"application/json","accept-encoding":"gzip","connection":"Keep-Alive","host":"localhost:8321","user-agent":"okhttp/4.8.1"},"body":"<binary body (0 bytes)>"}

```

EmmaLRussell reviewed (approved):
This does seem to be working for me after all, not sure how I broke it before!
